src/components/CharacterCard.js

```diff
@@ -1,5 +1,12 @@
 import React from "react";
+import { Container, Card, CardTitle } from 'reactstrap';
 
-export default function CharacterCard() {
-  return <span>todo: character</span>;
+export default function CharacterCard(props) {
+  return(
+    <Container>
+      <Card color='success' className='cardContainer' key={props.id}>
+        <CardTitle>{props.name}</CardTitle>
```

Comment:
I don't remember - during our 1:1, I thought you had images showing for each character 😮 

Adding in images would help the feel of the website, for sure '' '